### ISSUE TITLE
[A11y]: InoSnackbar is still not fully accessible

### ISSUE THREAD

**Issue by Sl1nd** [inovex/elements#1199]
### Bug Report

As snackbars optionally require an user action it should be possible to add role 'alertdialog' in addition to an accessible name expressing the need for action. In Addition currently its not possible to close the snackbar using "esc".

### Possible Solution

- Adding the possibility to configure the role of the snackbar ('alert', 'alertdialog')
- Close the  snackbar pressing "Esc"
- Hide icon from Screenreader as it is not accessible

Solution should be implemented referred to https://www.w3.org/WAI/ARIA/apg/patterns/alertdialog/examples/alertdialog/ as well as https://www.w3.org/WAI/ARIA/apg/patterns/alert/examples/alert/ to make all variants of ino-snackbar accessible.
